e2e/pages/app/facet-typeahead-list/facet-typeahead-list.testpage.component.ts

```diff
@@ -0,0 +1,56 @@
+import { Component } from '@angular/core';
+import { Facet, FacetEvent, FacetSelect, FacetDeselect, FacetDeselectAll } from '../../../../dist/lib/index.js';
+import { Observable } from 'rxjs/Observable';
+import { Observer } from 'rxjs/Observer';
+import 'chance';
+
+@Component({
+    selector: 'app',
+    templateUrl: './facet-typeahead-list.testpage.component.html'
+})
+export class FacetTypeaheadListPageComponent {
+
+    facets: Observable<Facet[]>;
+    suggestions: Facet[] = [];
+
+    private users: Facet[] = [];
```

Comment:
In several of these test page classes there are private instance variables (I understand they may be like this in the plunker which we should probably change at some point) but all variables referenced in the HTML file of a component should be public. I think the radio buttons and checkbox classes also have some variables marked as private.